content/docs/reference/glossary.md

```diff
@@ -24,6 +24,10 @@ A unique identifier used to authenticate a user or a calling program to an API.
 
 <a id="branches-coming-soon/"></a>
 
+### Availability checker
+
+A periodic load generated by the Control Plane to determine if a compute can start and read and write data. The availability checker queries a system database without accessing user data. Control Plane logic controls how often the availability checker runs to avoid too many concurrent availability checks when computes are started simultaneously.
```

Comment:
this change lacks the reference to the name of the operation that a user will see in the console in the operations log: `check_availability`

Not sure the amount of concurrent availability checks is relevant for the user facing docs.